TROUBLESHOOTING.md

```diff
@@ -0,0 +1,66 @@
+# Troubleshooting
+
+## Unexpected changes in the lockfile
+
+Running this migrator should not cause changes to your npm/yarn/pnpm lockfile, especially to existing dependencies?
+
+```
+npx ember-addon-migrator extract-tests --reuse-existing-versions
+```
+
+Using `--reuse-existing-versions` should help lock down the dependencies to the versions you have in your existing project.
+
+After this, do a `git diff` to inspect what new dependencies the migrator has added. Are these dependencies currently used in your project? If so, they should match your current project using the `--reuse-existing-versions` flag.
+
+If there are new dependencies the migrator has added, they might be needed for v2 addons.
+
+See the following for reference: https://github.com/embroider-build/embroider/blob/main/docs/porting-addons-to-v2.md#part-4-convert-addon-to-v2
+
+
+## Typescript errors
+
+If you encounter typescript errors that you haven't seen before, it's likely that the some of the following has been busted while running the migrator:
+
+1. `references` in tsconfig
+2. `extends` in tsconfig.compiler-options.json
+3. types folders (the migrator overrides the types folder types needed for a v2 addon, it's possible your original types have been deleted). 
+
+[//]: # (TODO: remove when these are fixed)
+
+Related Issues (these are currently TODO)
+https://github.com/NullVoxPopuli/ember-addon-migrator/issues/65
```

Comment:
Will wait because Simon is working on 65/66:

https://github.com/NullVoxPopuli/ember-addon-migrator/pull/75

Reply:
Moving this to draft for now